crates/swc_ecma_transforms_base/src/resolver/mod.rs

```diff
@@ -549,6 +549,18 @@ impl<'a> VisitMut for Resolver<'a> {
 
             let old = child.ident_type;
             child.ident_type = IdentType::Binding;
+            {
+                let params = e
+                    .params
+                    .iter()
+                    .filter(|p| !p.is_rest())
+                    .flat_map(find_pat_ids)
```

Comment:
@kdy1 BTW, do you remember why excluding rest parameter is necessary here?

Reply:
Not sure. I think I should write more comments...